x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/RequestInfo.java

```diff
@@ -14,7 +14,10 @@
 public class RequestInfo {
     public static final String CLI = "cli";
     private static final String CANVAS = "canvas";
-    public static final List<String> CLIENT_IDS = Arrays.asList(CLI, CANVAS);
+    public static final String ODBC_32 = "odbc32";
+    private static final String ODBC_64 = "odbc64";
+    public static final List<String> CLIENT_IDS = Arrays.asList(CLI, CANVAS, ODBC_32, ODBC_64);
```

Comment:
Really minor: This and below can be Immutable sets, so the `contains()` we call on them are faster.
Not an issue for such small collections, but more future-proof and since we only use them for the `contains()` makes more sense to be sets instead of lists.